src/containers/SearchPage/components/form/CheckboxSelector.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import { ToggleItem } from '@ndla/ui/lib/Filter';
+import { ChangeEvent, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface Props {
+  checked: boolean;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  name: string;
+}
+
+const CheckboxSelector = ({ name, checked: checkedProp, onChange: onChangeProp }: Props) => {
```

Comment:
Navnet på komponenten tyder på at den er en generisk komponent, men på linje 34 ser du at den har en hardkoda id for revisjonslogg. Kanskje du skal sette id til checkedProp for å gjøre den meir generisk?

Reply:
Sant! Lager en dynamisk id, sånn at den kan gjenbrukes uten feil 😄